### ISSUE TITLE
Ecommerce: cart page display breaks at certain widths

### ISSUE THREAD

**Issue by briangrossman** [mitodl/mitxonline#493]
### Steps to Reproduce

- Navigate to the checkout cart for a course: `/cart/`
- Change the width of the window

### Expected Behavior

- The name of the course should remain in the course card and display properly

### Actual Behavior

- The display of the course card breaks (see image below) are widths approximately 991px to 768px

### Screenshot or Screencast

![image](https://user-images.githubusercontent.com/21342568/160465957-b30c65b6-165c-44a6-a166-92e1ab32503f.png)

**Comment by jkachel** [comment]
It seemed like the best way to resolve this was to move the breakpoint at which it switches over from "mobile" to "full-width" - this is in branch jkachel/493-fix-cart-item-width-css now. This fixes the order item cards and fixes some other layout stuff in the summary card (see extra-small discount code input field - that whole bit will likely be a mess too if there's a code applied). I did have a question re: layout here though. I also opted to have it set so that the cart items appear before the order summary card on the page. See screenshot:
![image](https://user-images.githubusercontent.com/945611/160659237-8dd0d8e6-1d23-4cf6-87d1-561a0cfffefc.png)

I'm bringing this up because, in mobile, it swaps those around - the cart summary appears first (per https://projects.invisionapp.com/share/KH11D3ZN53GC#/screens/462972599 ). However, this is an in-between state (literally, mobile uses the "sm" breakpoint, this is using the "md" one, and the normal full-width view is "lg") so there's not a layout in InVision for it. I think it works a bit better this way on larger-screened devices, given your hands are probably going to be around the bottom of the device on a tablet-style device (and that, in landscape mode, you get the full-width layout). (FWIW, the header is also somewhat different; on mobile, you don't get the "You are about to purchase" header text either.) 

Thoughts? Also tagging @rhysyngsun and @annagav for input. It's pretty easy to eliminate the in-between md state if that's the better option. 

(Also: images are broken on my local copy. The boxes in the DOM take up space regardless of whether the image loads or not, though.)
